hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/ReferenceDataResource.kt

```diff
@@ -26,20 +27,19 @@ import uk.gov.justice.digital.hmpps.prisonersearch.search.services.ReferenceData
 class ReferenceDataResource(private val referenceDataService: ReferenceDataService) {
   @Operation(
     summary = "*** BETA *** Reference data search",
-    description = """
-      BETA endpoint - reference data returned reflects the data assigned to prisoners rather than all the possible
+    description = """BETA endpoint - reference data returned reflects the data assigned to prisoners rather than all the possible
```

Comment:
swagger issue - if start with a newline then it things the description is code and puts it in purple instead 🤷